sample-keys/README.md

```diff
@@ -0,0 +1,127 @@
+# TLS and mTLS for Tornjak
+
+In order to enable TLS and mTLS connection for Tornjak, a server must be configured with certificate and key information. This document outlines the following steps to try out the feature:
+
+1. [Create the relevant TLS/mtls files](#create-the-relevant-tls-files)
+2. [Configure the Tornjak server](#configure-the-tornjak-server)
+3. [Making calls to Tornjak Backend](#making-calls-to-tornjak-backend)
+
+## Create the relevant TLS/mTLS files
+
+The relevant certificates and keys first require a CA to sign a certificate. 
+
+Note that sample certificates and keys are included in publicly provided image `ghcr.io/spiffe/tornjak-backend` within the `sample-keys` directory. The below steps are available to OPTIONALLY create your own local certificates and keys. 
+
+<details><summary><b> 🔴 [Click] Create your own CA and certificate/key pair</b></summary>
+
+### Create a CA
+
+Run `./create-ca.sh` to create a CA. It will put the necessary cert and key files in the `CA/` directory. 
+
+### Signing a cert
+
+Certificates are required for TLS and mTLS connectiosn with the Tornjak server. To create and sign a certificate run `./create-cert.sh <domain name> <name>`. 
+
+For example, to create a certificate to be run at the local host domain name, we can run: 
+
+```
+./create-cert.sh localhost client
+```
+
+which will create `client.key` and `client.crt` files that represent the key/cert pair to configure the client. 
+
+----
+
+</details>
+
+## Configure the Tornjak server
+
+The Tornjak server configuration has separate settings for TLS and mTLS connection. For full Tornjak server configuration, see [the Tornjak server configuration document](../docs/config-tornjak-server.md). 
+
+Please click to expand below sections for configuring each connection type.  These sample paths assume that the relevant certificates and keys are located within `sample-keys/` as is in the default container image. To use custom keys, you will need to edit the paths. 
+
+<details><summary><b> 🔴 [Click] TLS configuration</b></summary>
+
+The TLS configuration requires a certificate and key pair. This is formatted like so:
+
+```
+server {
+  ...
+  tls {
+    enabled = true
+    port = 20000                 # container port for TLS connection
+    cert = "sample-keys/client.crt" # TLS cert
+    key = "sample-keys/client.key"  # TLS key
+  }
+  ...
+}
+```
+
+----
+
+</details>
+
+<details><summary><b> 🔴 [Click] mTLS configuration</b></summary>
+
+The mTLS configuration requires a certificate and key pair, along with a CA certificate to verify client requests. The mTLS configuration is formatted like so: 
+
+```
+server {
+  ...
+  mtls {
+    enabled = true
+    port = 30000                  # container port for mTLS connection
+    cert = "sample-keys/client.crt"  # mTLS cert
+    key = "sample-keys/client.key"   # mTLS key
+    ca = "sample-keys/CA/rootCA.pem" # mTLS CA
+  }
+  ...
+}
+```
+
+----
+
+</details>
+
```

Comment:
Prior to starting the Tornjak server, we should deliver the appropriate keys and certs to Tornjak container. This can be done via secrets as described here: https://github.com/IBM/trusted-service-identity/blob/main/docs/spire-security.md

We can just describe a manual steps. Helm charts would handle the key and secret distributions when using helm.